src/RealAntennasProject/Physics.cs

```diff
@@ -79,7 +79,7 @@ public static float PointingLoss(double angle, double beamwidth)
             else return math.lerp(9, 10, (norm - 0.86f) / 1);
         }
         public static float PointingLoss(RealAntenna ant, Vector3 origin)
-            => (ant.CanTarget && ant.ToTarget != Vector3.zero) ? PointingLoss(Vector3.Angle(origin - ant.Position, ant.ToTarget), ant.Beamwidth) : 0;
+            => (ant.CanTarget && !ant.IsTracking) ? PointingLoss(Vector3.Angle(origin - ant.Position, ant.ToTarget), ant.Beamwidth) : 0;

```

Comment:
Just be careful here, as the pointingloss for a poorly defined angle (ie direction to target is somehow near 0,0,0) will produce bad results.  Unsure how much this particular library code is exercised now, especially with the conversion to jobs.

Reply:
Well, this won’t affect cases where ToTarget is _near_ zero, only those where it _is_ zero.
It seems unlikely that it would be zero in cases where it should be, since it is computed as difference in double precision where one of the terms is Position, which is PrecisePosition, which is a position in double precision (the other term of the difference, Target.transform.position, is regrettably in single precision, but the result, while perhaps garbage, shouldn’t come out to 0).

(The difference is then converted to single precision but this matters not. All that switching between precisions is neither a recipe for performance nor correctness, but let’s not get distracted.)

Reply:
Could be irrelevant if calculating garbage data anyway [what is ToTarget with a somehow broken/invalid target?].  Other source of potentially bad result is maybe no longer possible for reason you described [PrecisePosition is double].  Main indicator might be vessels & targets that are extremely far away from current origin [ie, focus on a vessel past Pluto, what are the calculations for vessels/targets in similar orbit near Earth?]